modules/loader/src/main/scala/com/snowplowanalytics/snowplow/postgres/config/LoaderConfig.scala

```diff
@@ -127,6 +129,11 @@ object LoaderConfig {
       }
     }
 
+    implicit val pathDecoder: Decoder[Path] =
+      Decoder.decodeString.emap { p =>
+        Path.fromString(p).toRight(s"Invalid path: $p")
```

Comment:
Are we sure `Path` doesn't provide any methods with clear error message?

Reply:
`fromString` is returning `Option`. I couldn't find any other method with error message.